gcp_variant_transforms/transforms/call_info_to_bigquery.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Union  # pylint: disable=unused-import
+
+import apache_beam as beam
+
+from gcp_variant_transforms.beam_io import vcf_header_io  # pylint: disable=unused-import
+from gcp_variant_transforms.libs import call_info_table_schema_generator
+from gcp_variant_transforms.libs import hashing_util
+
+
+class ConvertCallInfoToRow(beam.DoFn):
+  """Extracts call info from `VcfHeader` and converts it to a BigQuery row."""
+
+  def process(self, vcf_header):
+    # type: (vcf_header_io.VcfHeader) -> Dict[str, Union[int, str]]
+    for sample in vcf_header.samples:
+      call_id = hashing_util.generate_unsigned_hash_code(
+          vcf_header.file_path + sample, max_hash_value=pow(2, 63))
+      row = {
+          call_info_table_schema_generator.CALL_ID: call_id,
+          call_info_table_schema_generator.CALL_NAME: sample,
+          call_info_table_schema_generator.FILE_PATH: vcf_header.file_path
+      }
+      yield row
+
+
+class CallInfoToBigQuery(beam.PTransform):
+  """Writes call info to BigQuery."""
+
+  def __init__(self, output_table_prefix, append=False):
+    # type: (str, Dict[str, str], bool) -> None
+    """Initializes the transform.
+
+    Args:
+      output_table_prefix: The prefix of the output BigQuery table.
+      append: If true, existing records in output_table will not be
+        overwritten. New records will be appended to those that already exist.
+    """
+    self._output_table = ''.join([
```

Comment:
I am thinking about moving this logic to a separate module that it's job is to generate table name based on the table_prefix and suffixes for different output tables. We need this logic for the following tables:
* Per chromosome table
* Variants vs non-variants tables
* partitioned and clustered by calls
* and this samples table

I am thinking we can move all the suffix consts to that module (including TABLE_SUFFIX = '_call_info' in call_info_table_schema.py) to that module and also the logic of adding underscore characters between name segments. That way user can modify only one spot to change the naming logic if they wanted to.

Please let me know what you think.